tests/python/driver/tvmc/test_runner.py

```diff
@@ -96,3 +96,14 @@ def test_run_tflite_module__with_profile__valid_input(
     assert type(outputs) is dict
     assert type(times) is tuple
     assert "output_0" in outputs.keys()
+
+
+def test_run_tflite_module_on_opencl(tflite_opencl_compiled_module_as_tarfile):
+    # some CI environments wont offer TFLite, so skip in case it is not present
+    pytest.importorskip("tflite")
+
+    outputs, times = tvmc.runner.run_module(
+        tflite_opencl_compiled_module_as_tarfile, hostname=None, device="cl"
+    )
+    assert type(outputs) is dict
+    assert type(times) is tuple
```

Comment:
Can you add one more check, to see whether the output tensor was produced? (see line 98 for an example)

Reply:
Added the output check.